packages/amplify-category-api/src/index.ts

```diff
@@ -26,7 +26,7 @@ import { checkAppsyncApiResourceMigration } from './provider-utils/awscloudforma
 import { getAppSyncApiResourceName } from './provider-utils/awscloudformation/utils/getAppSyncApiName';
 import { getAPIResourceDir } from './provider-utils/awscloudformation/utils/amplify-meta-utils';
 import { configureMultiEnvDBSecrets } from './provider-utils/awscloudformation/utils/rds-secrets/multi-env-database-secrets';
-import { deleteConnectionSecrets, readDatabaseNameFromMeta } from './provider-utils/awscloudformation/utils/rds-secrets/database-secrets';
+import { deleteConnectionSecrets, getSecretsKey, getDatabaseName } from './provider-utils/awscloudformation/utils/rds-secrets/database-secrets';
```

Comment:
Looks 'getDatabaseName' is unused on this file.

Reply:
These sometimes slip through my eyes. Apparently, there is a VSCode shortcut `option or alt + Shift+ O ` to "organize imports" which I tried and it removes the unused imports from the file and also sorts them. Sample,
```
import {
  addApiWithBlankSchema, amplifyPush,
  createNewProjectDir, deleteDBInstance, deleteProject,
  deleteProjectDir, importRDSDatabase, initJSProjectWithProfile, RDSTestDataProvider, removeApi, removeRDSPortInboundRule, updateApiSchema
} from 'amplify-category-api-e2e-core';
import generator from 'generate-password';
import { setupRDSDatabase, verifyAmplifyMeta, verifyCompiledSchema, verifyRDSSchema } from '../rds-v2-test-utils';
```
Acknowledging that it is more useful to do this consistently and automatically across modified files via some pre-commit hook.
Just sharing some tip.

Reply:
Will take care of this in upcoming PR.